content/02.main-text.md

```diff
@@ -220,6 +220,10 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+We are also [exploring](https://github.com/manubot/manubot/pull/99 "Create a pandoc-manubot-cite filter for pandoc") whether we can implement citation-by-identifier as a Pandoc filter.
+This would allow Pandoc users to cite persistent identifiers as part of their existing Pandoc workflows, without requiring them to adopt other aspects of Manubot.
+Since Pandoc already has a large user base, who convert to and from from a broad range of formats for a broad range of applications, a cite-by-identifier filter could help popularize the practice at an influential scale.
```

Comment:
We mention Pandoc above, but I wonder whether we need more context here about what Pandoc does and what we many by converting formats.  It may not convey how broadly useful Pandoc is.

I also prefer replacing one of the `broad range`s with a synonym.

Reply:
Rewrote this section, providing the much-needed context regarding pandoc